### ISSUE TITLE
Can USB-C port output video signal(DP Alt Mode)？

### ISSUE THREAD

**Issue by KrisCris** [zacmks/Hackintosh-Aero-15X#13]
Hi, 

Can your aero15x output video signal over DP ALT? (if yours' TB3 is wired to iGPU) 

Mine is the first gen Aero15 with TB3 port wired to integrated GPU, so I am sure it has that ability, but I had no idea how to make it working.

Also, I think this functionality do not rely on Thunderbolt since my Intel NUC supports that flawlessly without TB hotplug.

**Comment by zacmks** [comment]
Yes, usb c can output video. For Aero 15x V8 it does work, I do use it

**Comment by KrisCris** [comment]
How did you make that working? Any SSDTs required?
